### DISCUSSION TITLE
Global Smooth scrolling

### DISCUSSION THREAD

**Discussion by Dolbaeb777K** [hyprwm/Hyprland#6243] [General]
It just some idea.
I mean like macos/windows: https://www.smoothscroll.net/win/

**Comment by michelklaasen** [comment]
I second this idea. 

I used smoothscroll on windows, it's great. Especially when it's just one setting for the whole system, all applications.

I cannot find such an option in hyprland / wayland. Is it feasible?

Options I imagine would be necessary:
smooth scroll enable/disable
smooth scroll intensity (parameters to tweak smoothness)
smooth scroll exclude programs

**Reply by fufexan** [reply]
Looks like this emulates high-precision scroll wheels. Not sure how this is implemented, but I think it is feasible.
